convert2rhel/main.py

```diff
@@ -183,6 +179,26 @@ def main_locked():
     return 0
 
 
+# TODO(r0x0d): Better function name
+def _analyze_actions_results(results):
+    """Analyze the action results for failures
+
+    :param results: The action results from the framework
+    :type results: dict
+    """
+    failures = actions.find_actions_of_severity(results, "SKIP", level_for_raw_action_data)
+    if failures:
+        # The report will be handled in the error handler, after rollback.
+        loggerinst.critical("Conversion failed.")
+
+    # Print the assessment just before we ask the user whether to continue past the PONR
+    report.summary(
+        results=results,
+        include_all_reports=False,
+        disable_colors=logger_module.should_disable_color_output(),
+    )
```

Comment:
Move this out to be it's own function, doesn't have to do with analyze

Reply:
updated